lib/puppet/provider/user/hpux.rb

```diff
@@ -63,4 +75,43 @@ def password
       return ent.passwd
     end
   end
+
+  def trusted
+      trusted_sys = %x(/usr/lbin/getprpw root 2>&1)
+      if trusted_sys.chomp == "System is not trusted."
```

Comment:
The default behavior if getprpw isn't found or returns an error is to label the system "Trusted". Is that the desired default?